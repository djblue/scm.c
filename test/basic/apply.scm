(write (apply + '(1 2 3)))
(write (apply cons '(a b)))
(write (apply (lambda x x) '(+ 1 2 3)))
