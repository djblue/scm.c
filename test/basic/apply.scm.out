6
(a . b)
(+ 1 2 3)
